turbinia/workers/__init__.py

```diff
@@ -452,9 +452,9 @@ class TurbiniaTask:
 
   # The list of attributes that we will persist into storage
   STORED_ATTRIBUTES = [
-      'id', 'job_id', 'start_time', 'last_update', 'name', 'evidence_name',
-      'evidence_id', 'request_id', 'requester', 'group_name', 'reason',
-      'group_id'
+      'id', 'job_id', 'job_name', 'start_time', 'last_update', 'name',
```

Comment:
Why not just use ```name``` instead? Do we need the Job name or is the Task name sufficient?

Reply:
Yeah it would be harder to do it by `task_name` because as of current there isn't any `/api/tasks` endpoint like the `/api/jobs` endpoint which lists available jobs. The only workaround I was able to think of for filtering by task status instead is grabbing all available task names from iterating across all available tasks then building the select filter from there.

With the `/api/jobs` I just query available jobs into the list then pass it down to task-list to filter. We can consider a `/api/tasks` endpoint in the future though if it makes sense for filtering